components/snackbar/snackbar.css

```diff
@@ -0,0 +1,35 @@
+.snackbar-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--color-white);
+  position: relative;
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
+.snackbar {
+  width: 35rem;
+  height: 10rem;
+  height: 5rem;
+  margin: 0.1rem;
+  padding: 1rem 1.5rem;
+  background-color: var(--color-black);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 1%;
+}
+
+.fas {
```

Comment:
using this might give errors in future ...... but if you are sure that you won't use it...then it's fine ....

Reply:
It's a standard I defined. Will update it for future uses using a specific selector.